### PR TITLE
Office365: refactor the prerequisites

### PR DESCRIPTION
**Pull request by squioc**
Explain the reason of the prerequisites

### CODE REVIEW

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md [hunk 1]
```diff
@@ -25,10 +25,8 @@ Sekoia has developed an automatical setup for collecting Microsoft Office 365 lo
 
 ### Prerequisite
 
-- Have access to the Operations Center in order to create an intake
-- Be Administrator of the MS O365
-- An Azure Premium P1 or Azure Premium P2 license
-- Configure the MS O365 logging in the GUI
+- Have access to the Operations Center and be Administrator of the MS 0365 in order to create an intake
+- Have an Azure Premium P1 or Azure Premium P2 license and configure the MS O365 logging in the GUI to generate logs
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/cloud_and_saas/o365.md:29]:
```suggestion
To create an intake:

- access to the SEKOIA.IO XDR Operations Center 
- Microsoft Office 365 Administrator permissions

To generate logs:

-  Azure Premium P1 or Azure Premium P2 license
- Configure the Microsoft Office 365 logging in [SEKOIA.IO XDR](https://app.sekoia.io/operations/intakes/new)
```

**Review by PierrickV — APPROVED** [review]

**Review by otetard — APPROVED** [review]

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md (hunk 1 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/cloud_and_saas/o365.md]: applied

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md [hunk 2]
```diff
@@ -19,19 +19,38 @@ Microsoft Office 365 API logs four categories of logs:
 - Audit.General
 
 
-## Transport to SEKOIA.IO via API
+## Collect events
 
-Sekoia has developed an automatical setup for collecting Microsoft Office 365 logs from its dedicated API.
+According your needs, Sekoia allows to ways for collecting Micosoft Office 365:
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/cloud_and_saas/o365.md:24]:
```suggestion
According to your needs, SEKOIA.IO XDX allows two ways of collecting Microsoft Office 365 events:
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md [hunk 3]
```diff
@@ -19,19 +19,38 @@ Microsoft Office 365 API logs four categories of logs:
...
+
+- Through the Office 365 Management API (the recommendating way)
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/cloud_and_saas/o365.md:26]:
```suggestion
- Through the Office 365 Management API (the recommended way)
```

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md [hunk 4]
```diff
@@ -19,19 +19,38 @@ Microsoft Office 365 API logs four categories of logs:
...
+
+
+### From the Management API
+
+#### Prerequisites
+
+You will need the following prerequisites to create an intake:
+
+- access to the SEKOIA.IO XDR Operations Center 
+- be a Microsoft Office 365 Administrator to consent the interconnection
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/cloud_and_saas/o365.md:37]:
```suggestion
- be a Microsoft Office 365 Administrator to consent to the interconnection
```

**Review by PierrickV — APPROVED** [review]

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md (hunk 2 above)
Comment (gbossert) [reply docs/xdr/features/collect/integrations/cloud_and_saas/o365.md:24]: SEKOIA.IO XDR

**Comment by gbossert** [comment]
I suggest to focus on the API based collection as the main solution and provide as part of a bonus/annex the description of the  Azure Event Hub approach.

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md (hunk 2 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/cloud_and_saas/o365.md]: This section was deleted

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md (hunk 3 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/cloud_and_saas/o365.md]: This section was deleted

File: docs/xdr/features/collect/integrations/cloud_and_saas/o365.md (hunk 4 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/cloud_and_saas/o365.md]: applied

**Comment by squioc** [comment]
@gbossert I moved the EventHub approach in a dedicated file

**Review by PierrickV — APPROVED** [review]

**Pull request merged**